omod-common/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/MainSubResourceController.java

```diff
@@ -79,8 +82,17 @@ public SimpleObject getAll(@PathVariable("resource") String resource, @PathVaria
 	        @PathVariable("subResource") String subResource, HttpServletRequest request, HttpServletResponse response)
 	        throws ResponseException {
 		baseUriSetup.setup(request);
-		RequestContext context = RestUtil.getRequestContext(request, response);
 		SubResource res = (SubResource) restService.getResourceByName(buildResourceName(resource) + "/" + subResource);
+		RequestContext context = RestUtil.getRequestContext(request, response);

```

Comment:
What 'get'? MainSubResourceController doesn't have 'get' method, only 'getAll' to fetch multiple objects and 'retrieve' to get single object of subresource with specified UUID.